services/agents/registry.go

```diff
@@ -47,6 +47,9 @@ const (
 	prometheusSubsystem = "agents"
 )
 
+var defaultActionTimeout = ptypes.DurationProto(10 * time.Second)
```

Comment:
We typically use
```go
var (
  var1 = ...
  var2 = ...
)
```

And comments would be nice too.

But that's a minor thing.